tests/conftest.py

```diff
@@ -14,6 +14,7 @@
 """
 Pytest configuration file for PennyLane test suite.
 """
+# pylint: disable=unused-import
```

Comment:
Is the unused import going to be used in a subsequent PR? If not, it could just be removed instead of disabling the pylint warning.

Reply:
The imports are not used as recognized by pylint, but they are used implicitly to check for import errors. For example, the `dask_support_fixture`  imports `dask` but then only checks whether there is an import error.

Reply:
we should probably start using importlib (eg. `importlib.util.find_spec("dask")`) but we can do it in another PR

Reply:
Yep, seems reasonable! But definitely something that should be changed across the board in a single PR.